errbot/backends/slack.py

```diff
@@ -838,6 +838,26 @@ def build_reply(self, mess, text=None, private=False):
             response.to = mess.frm.room if isinstance(mess.frm, RoomOccupant) else mess.frm
         return response
 
+    def add_reaction(self, mess, reaction):
+        return self._react('reactions.add', mess, reaction)
+
+    def remove_reaction(self, mess, reaction):
+        return self._react('reactions.remove', mess, reaction)
+
+    def _react(self, api_call, mess, reaction):
+        result = self.sc.api_call(api_call,
+                                  channel=mess.to.channelid,
+                                  timestamp=mess.extras['ts'],
+                                  name=reaction)
```

Comment:
I would rather make use of the [self.api_call()](https://github.com/r3m0t/errbot/blob/64b89797abad251fcd3eee4f7f9526fe71d18e62/errbot/backends/slack.py#L303-L334) helper function here.